sbvr-libs.ometajs

```diff
@@ -51,7 +51,9 @@ SBVRLibs.AddVocabulary = function(vocabulary, baseSynonym) {
 };
 
 var formatFactType = function(factType) {
-	return _.map(factType, 1).join(' ');
+	return factType.map(function(v) {
+		return v[1];
+	}).join(' ');
```

Comment:
Any reason not to write this as:

```suggestion
	return factType.map((v) => v[1]).join(' ');
```

?

Reply:
You can't use `=>` or any syntax that is not es5 compliant because it all gets parsed by the ometajs parser that never had newer js syntax added

Reply:
As I was reminded when I tried precisely what you wrote and it failed to compile :P